### PR TITLE
use number as table id's - #1126

### PR DESCRIPTION
**Pull request by proddy**

### CODE REVIEW

**Comment by proddy** [comment]
I used numeric IDs where possible in the Table. Makes no real difference.

File: interface/src/project/SettingsScheduler.tsx [hunk 1]
```diff
@@ -52,15 +52,7 @@ import { useI18nContext } from 'i18n/i18n-react';
...
-  let result = '';
-  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-  const charactersLength = characters.length;
-  let counter = 0;
-  while (counter < 4) {
-    result += characters.charAt(Math.floor(Math.random() * charactersLength));
-    counter += 1;
-  }
-  return result;
+  return Math.floor(Math.random() * (Math.floor(200) - 100) + 100);
```
Comment (MichaelDvP) [inline interface/src/project/SettingsScheduler.tsx:55]: This seems a bit strange to me: floor(200) is always 200, but i think it willalways generate a number between 100 and 200.

**Review by MichaelDvP — APPROVED** [review]

File: interface/src/project/SettingsScheduler.tsx (hunk 1 above)
Comment (proddy) [reply interface/src/project/SettingsScheduler.tsx:55]: yes I wanted a number between 100 and 200. Math.random() returns a floating point. I would probably work without the floor(200). I should test it in a small program and use 'node prog.js' to see the output.

Comment (MichaelDvP) [reply interface/src/project/SettingsScheduler.tsx:55]: I think `Math.floor(Math.random() * 100 + 100)` is the same, or `Math.floor(Math.random() * 100) + 100`

**Pull request merged**